Library/Homebrew/download_strategy.rb

```diff
@@ -830,6 +838,10 @@ def stage
     safe_system(*args)
   end
 
+  def source_modified_time
+    Time.parse Utils.popen_read("fossil", "info", "tip", "-R", cached_location.to_s)[/^parent: *\h* (.*)$/, 1]
```

Comment:
@UniqMartin agree on regex.

Thanks a lot about `parent` and `uuid` stuff, I can't explain why I wrote `parent` instead of `uuid`. It's something wrong with me.